### PR TITLE
Add recipe for h3pandas

### PR DESCRIPTION
**Pull request by DahnJ**
Adding a recipe for https://github.com/DahnJ/H3-Pandas

Tests run locally using `CONFIG=linux64 ./.scripts/run_docker_build.sh`

I can see that the other platforms are failing. I will work on repairing those soon.

### CODE REVIEW

**Comment by DahnJ** [comment]
This is now ready for review

File: recipes/h3pandas/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dopplershift) [inline recipes/h3pandas/LICENSE:1]: The `LICENSE` is already included in the PyPI tarball, so it shouldn't be manually included here.

**Review by dopplershift — CHANGES_REQUESTED** [review]
This is in really good shape. Just remove the unneeded license file since that will be picked up from the tarball (and stay in sync with upstream).

File: recipes/h3pandas/LICENSE (hunk 1 above)
Comment (DahnJ) [reply recipes/h3pandas/LICENSE]: Thanks for the review, I removed the `LICENSE` file.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**